harness/determined/exec/prep_container.py

```diff
@@ -55,14 +55,14 @@ def trial_prep(info: det.ClusterInfo, cert: certs.Cert) -> None:
 
 
 def do_rendezvous(info: det.ClusterInfo, cert: certs.Cert) -> None:
-    # Even though container_id is not part of the ClusterInfo API, we still depend on it in all
+    # Even though resources_id is not part of the ClusterInfo API, we still depend on it in all
```

Comment:
Was this the method where we were supposed to send back the IP address of compute node's high speed interconnect to the master?  If so, wasn't the "get" going to be changed to a "post" or did I misunderstand?

Reply:
new API to use is in this PR https://github.com/determined-ai/determined/pull/3694